src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1203,11 +1203,13 @@ private ITupleExpression CreateBoundTupleExpressionOperation(BoundTupleExpressio
 
         private IInterpolatedStringExpression CreateBoundInterpolatedStringExpressionOperation(BoundInterpolatedString boundInterpolatedString)
         {
-            Lazy<ImmutableArray<IInterpolatedStringContent>> parts = new Lazy<ImmutableArray<IInterpolatedStringContent>>(() =>
-                boundInterpolatedString.Parts.SelectAsArray(interpolatedStringContent => CreateBoundInterpolatedStringContentOperation(interpolatedStringContent)));
             SyntaxNode syntax = boundInterpolatedString.Syntax;
             ITypeSymbol type = boundInterpolatedString.Type;
             Optional<object> constantValue = ConvertToOptional(boundInterpolatedString.ConstantValue);
+            Lazy<ImmutableArray<IInterpolatedStringContent>> parts = new Lazy<ImmutableArray<IInterpolatedStringContent>>(() =>
+                boundInterpolatedString.Parts.IsDefaultOrEmpty ?
```

Comment:
As Aleksey would say, are we sure that this can be default, or would that be an error condition?